python/tvm_ffi/__init__.py

```diff
@@ -95,3 +95,9 @@
     "use_raw_stream",
     "use_torch_stream",
 ]
+
+for _name in __all__:
+    try:
+        setattr(globals()[_name], "__module__", "tvm_ffi")
+    except AttributeError:
+        pass
```

Comment:
@tqchen I will need to normalize `__module__` of objects like `core.Object` to `tvm_ffi.Object` so that the "as method XXX" part can find the method properly (see screenshot). Let me know if this makes sense!

<img width="792" height="261" alt="Image" src="https://github.com/user-attachments/assets/b22c27c1-6f5a-4f93-b849-de02493168b2" />